grpc-native/src/main/java/org/ballerinalang/net/grpc/GrpcConstants.java

```diff
@@ -277,4 +277,25 @@ private GrpcConstants() {
     //context message field name constants
     public static final String CONTENT_FIELD = "content";
     public static final String HEADER_FIELD = "headers";
+
+    // Compiler plugin constants
+    public static final String GRPC_ANNOTATION_NAME = "ServiceDescriptor";
+    public static final String BALLERINA_ORG_NAME = "ballerina";
+    public static final String GRPC_PACKAGE_NAME = "grpc";
+
+    // Diagnostic error messages and IDs
+    public static final String UNDEFINED_ANNOTATION_MSG = "undefined annotation: ";
```

Comment:
Error messages and error codes are coupled, right? We can have an enum? ([graphql](https://github.com/ballerina-platform/module-ballerina-graphql/blob/5549699ae7f15f4abf498d3694fe256b23c31546/graphql-compiler-plugin/src/main/java/io/ballerina/stdlib/graphql/compiler/PluginConstants.java#L40))

Reply:
Yeah, enums would be better.

Reply:
Fixed in herehttps://github.com/ballerina-platform/module-ballerina-grpc/pull/192/commits/6c60129213e867a866d25b3c4afd3e48922812bb